### PR TITLE
Meshmodel Browser

### PR DESCRIPTION
**Pull request by smartdev09**
**Notes for Reviewers**

This PR fixes #

https://github.com/meshery/meshery/assets/31584129/72f5ef12-a60f-4c74-83e8-00aba8de8e97

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/api/meshmodel.js [hunk 1]
```diff
@@ -66,9 +66,9 @@ export async function getVersionedComponentFromModel(
   );
 }
 
-export async function getComponentsDetailWithPageSize(page=1, pageSize="all") {
+export async function getComponentsDetailWithPageSize(page=1, pageSize="all", sortOrder) {
   return await promisifiedDataFetch(
-    `api/meshmodels/components?page=${page}&pagesize=${pageSize}`
+    `api/meshmodels/components?page=${page}&pagesize=${pageSize}&order=${encodeURIComponent(sortOrder)}`
```
Comment (leecalcote) [inline ui/api/meshmodel.js:71]: @UsmanDev09 why are you not using `sort` as a querystring parameter?

From the REST API docs: 
```
?sort={[asc/desc]} Default behavior is asc
```

// @nebula-aac

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/api/meshmodel.js (hunk 1 above)
Comment (leecalcote) [reply ui/api/meshmodel.js:71]: @UsmanDev09 can you finish this now?

File: ui/components/DuplicatesDataTable.js [hunk 2]
```diff
@@ -15,9 +15,9 @@ const DuplicatesDataTable = ({ view, rowData, classes }) => {
   const [, setSearchText] = useState(null);
   const [rowsPerPage, setRowsPerPage] = useState(10);
 
-  const kind = rowData[0].props?.children?.props?.children;
-  const model = rowData[3].props?.children?.props?.children;
-  const version = rowData[1].props?.children?.props?.children;
+  const kind = rowData[0]?.props?.children?.props?.children;
+  const model = rowData[3]?.props?.children?.props?.children;
+  const version = rowData[1]?.props?.children?.props?.children;
```
Comment (leecalcote) [inline ui/components/DuplicatesDataTable.js:20]: @UsmanDev09 Please reference this by name, not index. This code is fragile.

@sudhanshutech will you take over this PR, please?

File: ui/api/meshmodel.js (hunk 1 above)
Comment (UsmanDev09) [reply ui/api/meshmodel.js:71]: @leecalcote  Yes, I can

**Comment by leecalcote** [comment]
// @aabidsofi19

**Pull request merged**